includes/classes/order.php

```diff
@@ -770,7 +770,7 @@ function create_add_products($zf_insert_id, $zf_mode = FALSE) {
           }
           $stock_values = $db->Execute($stock_query_raw, false, false, 0, true);
         } else {
-          $stock_values = $db->Execute("select * from " . TABLE_PRODUCTS . " where products_id = '" . zen_get_prid($this->products[$i]['id']) . "'", false, false, 0, true);
+          $stock_values = $db->Execute("select products_quantity, product_is_always_free_shipping from " . TABLE_PRODUCTS . " where products_id = '" . zen_get_prid($this->products[$i]['id']) . "'", false, false, 0, true);
```

Comment:
Given that `'products_attributes_filename'` is also used on line 780, I think the query should also include `'' as products_attributes_filename`. But I suppose 780 could be rewritten too.